_includes/contact_form.html

```diff
@@ -168,6 +168,13 @@
     document.addEventListener('DOMContentLoaded', doNotAnswer);
 </script>
 
+<!--Banner for TTP - Remove when ready-->
+<div class="usa-alert usa-alert--info margin-bottom-4">
+  <div class="usa-alert__body">
+    <p class="usa-alert__text">New York Residents are no longer eligible to apply for or renew membership in U.S. Customs and Border Protection (CBP) Trusted Traveler Programs. Your refund will be processed automatically. For additional information, please visit <a href="https://www.cbp.gov/newsroom/national-media-release/new-york-residents-no-longer-eligible-apply-or-renew-trusted" target="_blank">the CBP media release</a>.</p>
```

Comment:
Should we put this in the `en.yml`?

Reply:
In this case, I don't think so because we are not changing anything in the YMLs' content, but rather creating a temporary supplement to the `contact_form.html` without needing to touch on YMLs. If this doesn't make sense, let me know

Reply:
I meant more like, we try to have all user-facing strings live in the YML files so that we can automatically use the translated versions if they are available, and this is a new string so I'd put it there.

But since this is supposed to be temporary, it may be fine as-is